packages/grpc-js/src/subchannel.ts

```diff
@@ -309,7 +309,8 @@ export class Subchannel {
         };
         connectionOptions.servername = sslTargetNameOverride;
       } else {
-        const authorityHostname = splitHostPort(targetAuthority)?.host ?? 'localhost';
+        const authorityHostname =
+          splitHostPort(targetAuthority)?.host ?? 'localhost';
```

Comment:
This is just an incidental formatting change that came from formatting the rest of the file.